src/config/field-overrides/dataSet.js

```diff
@@ -52,39 +54,149 @@ class RenderAsTabsSettings extends React.Component {
+            display: 'flex',
+            flexDirection: 'column',
+            marginLeft: '16px'
+        };
         return <div>
-           <Checkbox
-            labelText={this.translate('render_as_tabs')}
-            value={props.value}
-            onChange={this.onRenderAsTabsChanged}
-        />
-            {props.value &&
-             <RadioButtonGroup
-                onChange={this.onDisplayOptionsChanged}
-                name="tabsDirection"
-                defaultSelected={
-                    (state.displayOptions && state.displayOptions.tabsDirection) || 'horizontal'  }
-            >
-                <RadioButton
-                    key='horizontal'
-                    value='horizontal'
-                    label={this.translate('horizontal')}
-                    style={{margin: '10px'}}
-                />
-                <RadioButton
-                    key='vertical'
-                    value='vertical'
-                    label={this.translate('vertical')}
-                    style={{margin: '10px'}}
+            <div>
+                <Checkbox
+                    labelText={this.translate('render_as_tabs')}
+                    value={props.value}
+                    onChange={this.onRenderAsTabsChanged}
                 />
-            </RadioButtonGroup>}
+                {props.value &&
+                    <RadioButtonGroup
+                        onChange={this.onDisplayOptionsChanged}
+                        name="tabsDirection"
+                        defaultSelected={
+                            (state.displayOptions && state.displayOptions.tabsDirection) || 'horizontal'}
+                    >
+                        <RadioButton
+                            key='horizontal'
+                            value='horizontal'
+                            label={this.translate('horizontal')}
+                            style={{margin: '10px'}}
+                        />
+                        <RadioButton
+                            key='vertical'
+                            value='vertical'
+                            label={this.translate('vertical')}
+                            style={{margin: '10px'}}
+                        />
+                    </RadioButtonGroup>}
+            </div>
+            <div>
+            <Checkbox
+                labelText={this.translate('add_custom_text')}
+                value={state.displayOptions && state.displayOptions.customText !== undefined}
+                onChange={this.onAddCustomTextChanged}
+            />
+
+            {state.displayOptions && state.displayOptions.customText &&
+                <RadioButtonGroup
+                    onChange={this.onCustomTextAlignmentChanged}
+                    name="customTextAlignment"
+                    defaultSelected={
+                        (state.displayOptions && state.displayOptions.customText && state.displayOptions.customText.align) || 'left'}
```

Comment:
do you want `left` or `center` to be the default? It seems with `onAddCustomTextChanged`, that the alignment was being set to center by default?

Reply:
THANK YOUUUUU